ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,4 @@
+If you have any questions on Theano, please ask your questions in either theano-user mailing (https://groups.google.com/forum/#!forum/theano-users) or 
+stackoverflow (http://stackoverflow.com/).
```

Comment:
"mailing" -> "mailing list"
I would say `[StackOverflow](http://stackoverflow.com/) with the "theano" tag` instead of "stackoverflow"